Makefile

```diff
@@ -0,0 +1,15 @@
+all: test
```

Comment:
👍  to all what @jbenet said: keep 'em simple, use them to document *how* to get a repo up an running (*which* things to run is the most important description here), not for heavy lifting (npm is fine for that in JS, other langs have their own tools).

@jbenet also: cool idea to add command help to `make all`.